README.md

```diff
@@ -466,54 +466,54 @@ client.delete_secret(API_KEY, 'my-secret-id')
 
 ### Retrieve a list of applications
 
 ```python
 response = client.application_v2.list_applications()
 ```
 
-Docs: [https://developer.nexmo.com/api/application.v2#listApplication](https://developer.nexmo.com/api/application.v2#listApplication?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library#retrieve-your-applications)
+Docs: [https://developer.vonage.com/api/application.v2#listApplication](https://developer.vonage.com/api/application.v2#listApplication?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library#retrieve-your-applications)
 
 ### Retrieve a single application
 
 ```python
 response = client.application_v2.get_application(uuid)
 ```
 
-Docs: [https://developer.nexmo.com/api/application.v2#getApplication](https://developer.nexmo.com/api/application.v2#getApplication?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library#retrieve-an-application)
+Docs: [https://developer.vonage.com/api/application.v2#getApplication](https://developer.vonage.com/api/application.v2#getApplication?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library#retrieve-an-application)
 
 ### Update an application
 
 ```python
 response = client.application_v2.update_application(uuid, answer_method='POST')
 ```
 
-Docs: [https://developer.nexmo.com/api/application.v2#updateApplication](https://developer.nexmo.com/api/application.v2#updateApplication?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library#update-an-application)
+Docs: [https://developer.vonage.com/api/application.v2#updateApplication](https://developer.vonage.com/api/application.v2#updateApplication?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library#update-an-application)
 
 ### Delete an application
 
 ```python
 response = client.application_v2.delete_application(uuid)
 ```
 
-Docs: [https://developer.nexmo.com/api/application.v2#deleteApplication](https://developer.nexmo.com/api/application.v2#deleteApplication?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library#destroy-an-application)
+Docs: [https://developer.vonage.com/api/application.v2#deleteApplication](https://developer.vonage.com/api/application.v2#deleteApplication?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library#destroy-an-application)
 
 ## Validate webhook signatures
 
 ```python
-client = nexmo.Client(signature_secret='secret')
+client = vonage.Client(signature_secret='secret')
 
 if client.check_signature(request.query):
   # valid signature
 else:
   # invalid signature
 ```
 
-Docs: [https://developer.nexmo.com/concepts/guides/signing-messages](https://developer.nexmo.com/concepts/guides/signing-messages?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library)
+Docs: [https://developer.vonage.com/concepts/guides/signing-messages](https://developer.vonage.com/concepts/guides/signing-messages?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library)
```

Comment:
This URL should be updated. It only works because Github will redirect it, but it should point to the correct URL